docs/cli/edgedb_instance/edgedb_instance_create.rst

```diff
@@ -21,6 +21,20 @@ instance and creating a corresponding credentials file in
 ``<edgedb_config_dir>`` on your machine.
 
 
+EdgeDB Cloud
+------------
+
+.. TODO: Cloud release
+.. Update this after Cloud has released
+
+Users with access to the EdgeDB Cloud beta may use this command to create a
+Cloud instance after logging in using :ref:`ref_cli_edgedb_cloud_login`.
+
+To create a Cloud instance, your instance name should be in the format
+``<github-org-name>/<instance-name>``. Cloud instance names may contain
```

Comment:
Refs "org-name": I don't think we already supported GitHub organizations - at this moment it is just personal GitHub accounts. But we will support organizations in the future I believe, so perhaps we could adopt something more general here?

Reply:
Hmm. OK. I was thinking about labeling it instead `github-identified`, but then I'd feel the need to circle back and tell users we only support the username for now but will also support org name in the future. That will need to be changed once we support org name, so I think I might as well just call it `github-username` for now. Once we roll out support for organizations, I will circle back and change it to `github-identifier` and explain that can be either username or org name.